modules/Forum/pages/forum/view_topic.php

```diff
@@ -281,25 +281,104 @@
 					'topic_reply_date' => date('U')
 				));
 
-				// Alerts
+				// Alerts + Emails
 				$users_following = $queries->getWhere('topics_following', array('topic_id', '=', $tid));
 				if(count($users_following)){
+					$users_following_info = array();
 					foreach($users_following as $user_following){
-						if($user_following->user_id != $user->data()->id && $user_following->existing_alerts == 0){
-							Alert::create(
-								$user_following->user_id,
-								'new_reply',
-								array('path' => ROOT_PATH . '/modules/Forum/language', 'file' => 'forum', 'term' => 'new_reply_in_topic', 'replace' => array('{x}', '{y}'), 'replace_with' => array(Output::getClean($user->data()->nickname), Output::getClean($topic->topic_title))),
-								array('path' => ROOT_PATH . '/modules/Forum/language', 'file' => 'forum', 'term' => 'new_reply_in_topic', 'replace' => array('{x}', '{y}'), 'replace_with' => array(Output::getClean($user->data()->nickname), Output::getClean($topic->topic_title))),
-								URL::build('/forum/topic/' . $tid . '-' . $forum->titleToURL($topic->topic_title), 'pid=' . $last_post_id)
-							);
-							$queries->update('topics_following', $user_following->id, array(
-								'existing_alerts' => 1
-							));
+						//if($user_following->user_id != $user->data()->id){
```

Comment:
Are the comments on these lines intentional?

Reply:
Wow, how did I not notice that. I commented them out while testing.